apps/docs/docs/reference/ui/atoms/Link.mdx

```diff
@@ -1,21 +1,44 @@
 ---
 description: Links allow users to easily navigate through web pages.
+keywords:
+  - UI library
+  - ecommerce
+  - react
 sidebar_custom_props: 
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Link___ae95650d3842207e2d790446c3418086.png
 ---
 
 import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
-# Link
+Link performs the same behavior of a regular anchor tag (`a`), but can be used as any HTML tag and component, including spacifics Frameworks link components.
 
-The `Link` component behaves as a regular anchor tag (`<a>`), but can be used as any HTML tag.
+## Overview
+
+```tsx live
+<div style={{ display: 'flex', columnGap: '16px', alignItems: 'center' }}>
+  <Link variant="default">Default</Link>
+  <Link variant="display">Display</Link>
+  <Link variant="footer">Footer</Link>
+  <Link variant="inline">Inline</Link>
+  <Link inverse={true}>Inverse</Link>
+</div>
+```
 
```

Comment:
Should we include this divisor between the sections?
```suggestion

---

```

Reply:
I based on the documentation of the button